apps/coscrad-frontend/src/components/nav-menu/nav-menu-presenter.tsx

```diff
@@ -23,17 +23,19 @@ export const NavMenuPresenter = ({ navItemInfos }: NavMenuPresenterProps): JSX.E
 
     return (
         <Box sx={{ display: 'flex' }}>
-            <IconButton
-                data-testid="nav-menu-icon"
-                id="basic-button"
-                color="secondary"
-                aria-controls={isOpen ? 'basic-menu' : undefined}
-                aria-haspopup="true"
-                aria-expanded={isOpen ? 'true' : undefined}
-                onClick={handleClick}
-            >
-                <MenuRoundedIcon />
-            </IconButton>
+            <Tooltip title="Menu">
```

Comment:
We could also consider "Open the menu", "Open menu", "Expand the menu", "Expand menu", or something like this.

Reply:
Here I like just "Menu" because it's concise.  I think even without the tooltip here people know now what the icon represents, and I think "Menu" is consistent with what I've seen in other interfaces.